### ISSUE TITLE
[DataCap Application] End-Use Load Profiles for the U.S. Building Stock

### ISSUE THREAD

**Issue by quxxMel** [Destore2023/MetaPathways-Bookkeeping#18]
### Version

1

### DataCap Applicant

quxxMel

### Project ID

1

### Data Owner Name

National Renewable Energy Laboratory

### Data Owner Country/Region

United States

### Data Owner Industry

Environment

### Website

https://www.nrel.gov/buildings/end-use-load-profiles.html

### Social Media Handle

Marcus.Bianchi@nrel.gov

### Social Media Type

Slack

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

2PiB

### Expected size of single dataset (one copy)

216TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

256TiB

### On-chain address for first allocation

f12avrdgtgb2lqdruxsno3qdksz7ow5b25gc5if7q

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
From breakthroughs in fundamental science to new clean technologies to integrated energy systems that power our lives, NREL researchers are transforming the way the nation and the world use energy.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
The U.S. Department of Energy (DOE) funded a three-year project, End-Use Load Profiles for the U.S. Building Stock, that culminated in this publicly available dataset of calibrated and validated 15-minute resolution load profiles for all major residential and commercial building types and end uses, across all climate regions in the United States. These EULPs were created by calibrating the ResStock and ComStock physics-based building stock models using many different measured datasets, as described here. This dataset includes load profiles for both the baseline building stock and the building stock with various energy efficiency, electrification, and demand flexibility upgrades applied.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Hong Kong

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
Tool: servers, lotus
Lotus is the primary tool we use to transform data. We have 3 servers to support us dealing with dataset. We also support transfer data to sps online/offline.
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
No
```

### Please share a sample of the data

```text
aws s3 ls --no-sign-request s3://oedi-data-lake/nrel-pds-building-stock/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Sporadic

### For how long do you plan to keep this dataset stored on Filecoin

More than 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Filmine, Big Data Exchange

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f03161261 NewYork
f03136895 Virginia
f03087446 Los Angeles
f03157902 Singapore
f03189917 Virginia
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by quxxMel** [comment]
@Destore2023 Hi allocator, can you sign my application?

**Comment by Destore2023** [comment]
Please fill in this form to complete your information. https://www.wenjuan.com/s/qAVFfuN/

**Comment by quxxMel** [comment]
@Destore2023 Hello, I have filled the form.

**Comment by Destore2023** [comment]
@quxxMel 
Got your form!
![image](https://github.com/user-attachments/assets/b0eaa1cc-fcff-4151-846d-e8e8233334c2)

Seems to be the first time this dataset has been stored in Filecoin network. Welcome to store data in filecoin.

![1](https://github.com/user-attachments/assets/c2670b33-bb58-4b50-97d3-730669913616)


When you want to use datacap to store data, you'd better to select sps in different regions. Also, please make your sps support retrieval! We will check it from time to time.

**Comment by quxxMel** [comment]
@Destore2023 Thank you. We'll do as you say.

**Comment by Destore2023** [comment]
@quxxMel ok. Will give you the first round support.

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@quxxMel Please check your sps with 0 retrieval success rate. Also you should focus on data distributing.

**Comment by quxxMel** [comment]
@Destore2023 Hello allocator, sp gave feedback that they met this problem. _announce message to https://cid.contact/ingest/announce: 429 Too Many Requests: Too Many Requests_
They are checking about it.

**Comment by Destore2023** [comment]
@quxxMel ok. Look forward to your improvements.

**Comment by Destore2023** [comment]
@quxxMel 
This is the response from spark team. https://github.com/filecoin-station/spark/issues/100
The current condition is that the spark team made a limit and your request was too fast, resulting in a 429 warning. You can try to pay attention to control the speed.

> @Destore2023 Hello allocator, sp gave feedback that they met this problem. _announce message to https://cid.contact/ingest/announce: 429 Too Many Requests: Too Many Requests_ They are checking about it.

**Comment by quxxMel** [comment]
@Destore2023 Thank you very much!
We invite more sps to join our storage. Please check it.
f01660795 Guangdong
f03192503 HongKong
f03214920 HongKong
f03066836 Hunan
f03231666 Sichuan

**Comment by Destore2023** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
Improvement in comparison with the last retrieval result. Glad to see your improvement on retrieval.


_Last time_
![image](https://github.com/user-attachments/assets/520ecb61-74a6-4ce4-8b1b-77ecb38f1056)
_This time_
![image](https://github.com/user-attachments/assets/96748d59-deb4-471e-8eeb-29f44c37394d)


But can you explain why sps sealed too much duplicate data?

**Comment by quxxMel** [comment]
@Destore2023 Hello allocator, can you help us check the report? The amount of _total deals sealed_ in report is more than datacap we received.

**Comment by quxxMel** [comment]
<img width="1363" alt="1" src="https://github.com/user-attachments/assets/db69609f-036c-4cf0-ad68-d3a6a6eeea2d">
<img width="1081" alt="2" src="https://github.com/user-attachments/assets/62416a57-d3a5-415a-8dfd-ce05bf0f0974">

For example
As shown in the picture, the total amount of datacap got by f03228358 is 217703302299648 bytes (198TiB).

But it shows that this sp has got 339.63TiB.
![image](https://github.com/user-attachments/assets/0fe63496-d594-4e58-9823-ac54c50a48c5)
 
 Please help us about checking it. Thank you!

**Comment by Destore2023** [comment]
@quxxMel Hello, thanks for your words.
There's some problem on the cid report about duplicate data. Someone has submitted this issue, and I will focus on the situation. Please don't worry about it.

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by Destore2023** [comment]
@quxxMel Do you confirm that f01975299 is in your sp list?

**Comment by quxxMel** [comment]
@Destore2023 Yes. We found a new sp and sent some data to him. I'm sorry I didn't let you know first.

**Comment by quxxMel** [comment]
@Destore2023 Hello allocator, there're two sps(f03233966, f03100009) who will not involve in our plan and support retrieval next step. New sps are joining us.
f01431043 Hongkong f03241837 Shanghai
We have confirmed that they support retrieval.

**Comment by Destore2023** [comment]
checker:manualTrigger
